types/pino-http/index.d.ts

```diff
@@ -39,4 +39,7 @@ declare module 'http' {
     interface IncomingMessage {
         log: Logger;
     }
+    interface ServerResponse {
```

Comment:
Instead of module augmentations, why not have local types that extend the base types? e.g.

```
import { IncomingMessage as IncomingMessageBase, ServerResponse as ServerResponseBase } from 'http';

interface IncomingMessage extends IncomingMessageBase {
    log: Logger;
}

interface ServerResponse extends ServerResponseBase {
    err?: Error;
}
```

This way, you don't pollute the global scope.

Reply:
Sure, I'm happy to do this in this PR if @jdforsythe is on board.  I'm a bit of a n00b at this so forgive the stupid question, but: would these interfaces sit under the `PinoHttp` namespace? I would think this would be a breaking change for users of this package. Or do they simply sit at the top level of this module?

Reply:
In fact I'm not sure how this change wouldn't be breaking for consumers, given the point of the package is to add a `log` property for use within http handlers. Presumably anyone using this package would have to change their handlers to reference the new interface rather than the `http` one?